documentation/modules/ROOT/partials/modules/all-connectors/frag-common-mbean-name.adoc

```diff
@@ -0,0 +1,28 @@
+// Common
+tag::common-snapshot[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,context=snapshot,server=_<{context}.server.name>_`.
+end::common-snapshot[]
+
+tag::common-streaming[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,context=streaming,server=_<{context}.server.name>_`.
+end::common-streaming[]
+
+
+// MongoDB
+tag::mongodb-snapshot[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,context=snapshot,server=_<{context}.server.name>_,task=_<task.id>_`.
+end::mongodb-snapshot[]
+
+tag::mongodb-streaming[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,context=streaming,server=_<{context}.server.name>_,task=_<task.id>_`.
+end::mongodb-streaming[]
+
+
+// SQL Server
+tag::sqlserver-snapshot[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,server=_<{context}.server.name>_,task=_<task.id>_,context=snapshot`.
+end::sqlserver-snapshot[]
+
+tag::sqlserver-streaming[]
+The *MBean* is `debezium.{mbean-name}:type=connector-metrics,server=_<{context}.server.name>_,task=_<task.id>_,context=streaming`.
+end::sqlserver-streaming[]
```

Comment:
Rather than use separate files to hold each of the 6 MBean name formats, I realized that I could _tag_ regions within a single file to specify the naming format. The `include` statement in the hosting connector file then specifies the region from which to pull the name format.